packages/php/blueprint/src/docs/json-examples/setOptions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "step": "setOptions",
```

Comment:
I think it should be `setSiteOptions` here

Reply:
Thank you!

Updated in [Rename setOptions to setSiteOptions](https://github.com/woocommerce/woocommerce/pull/49763/commits/3aa3825b6877e288ca65a99717e4e0b8158239b7)

I've also updated other examples files in [Update json example files](https://github.com/woocommerce/woocommerce/pull/49763/commits/66c0b5456585e6671b04a58313aa4f0a1ed40308) as they are outdated.